madgraph/various/banner.py

```diff
@@ -868,7 +868,7 @@ def append(self, line):
                 self.info['full_model_line'] = line
                 self.clean(remove_bef_last='import', keep_switch=True,
                         allow_for_removal=['generate', 'add process', 'add model', 'output'])
-                if cmds[1] == 'model' and len(cmds)>2:
+                if cmds[1] == 'model':
```

Comment:
why removing this condition? this is making the code to crash if this is not true?

Reply:
Hi Olivier,

I didn't touch this line, but maybe it's something new, I have never seen it.
But I can tell that in 3.x this line is there - so I'm going to add it.